package_installs.R

```diff
@@ -1,7 +1,9 @@
 # Repo to pull package data and metadata from.
 REPO <- 'http://ftp.osuosl.org/pub/cran'
 # Number of parallel installs.
-M <- 4
+M <- 64
+# Make use of all CPUs, it's linked to the GCB custom VM size.
```

Comment:
Nit:  Two sentences, or perhaps something to the effect of "Make use of all CPUs available to the custom GCB VM size we use."